### DISCUSSION TITLE
Saving model to file

### DISCUSSION THREAD

**Discussion by Ttimofeyka** [karpathy/llm.c#254] [General]
I may be wrong, but there is no writing to a file in the code and splitting the done model into checkpoints during training. Will it be added?

**Comment by Ttimofeyka** [comment]
@karpathy

**Comment by karpathy** [comment]
Yep no such thing exists. There is no reason to do this right now but we'll get there.
We're now doing multigpu bfloat16 runs, so we're doing well, probably not too far away.
